src/components/dialogs/network-modifications/two-windings-transformer/modification/two-windings-transformer-modification-dialog.js

```diff
@@ -226,7 +227,8 @@ const TwoWindingsTransformerModificationDialog = ({
                 ...getRatioTapChangerFormData({
                     enabled:
                         twt?.[RATIO_TAP_CHANGER]?.[ENABLED]?.value ??
-                        !!twtToModify?.ratioTapChanger,
+                        (!_.isEmpty(twt?.[RATIO_TAP_CHANGER]) ||
+                            !!twtToModify?.ratioTapChanger),
```

Comment:
These three lines of code are unreadeable/unmaintainable. Could you do the logic somewhere else, with an easy to read and understand algorithm ?

Reply:
done